README.md

```diff
@@ -200,6 +200,17 @@ Worker Properties file provided at startup. The Kafka Producer instances created
 can also be configured using a `producer.` prefix on the standard
 [Kafka Producer properties](https://kafka.apache.org/documentation/#producerconfigs).
 
+## Destination Topic Checking
+
+By default, Mirus checks that the destination topic exists in the destination Kafka cluster before
+stating to replicate data to it. This feature can be disabled by setting the
+[disable.destination.topic.checking](src/main/java/com/salesforce/mirus/config/SourceConfigDefinition.java#L66)
+config option to `true`.
```

Comment:
I think I prefer "positive" rather than "negative" feature flags: so `enable...`, defaulting to `true`.

Reply:
Same with the boolean flags within the code.